cameraview/src/main/res/values/attrs.xml

```diff
@@ -124,5 +124,7 @@
 
         <attr name="cameraExperimental" format="boolean" />
 
+        <attr name="cameraAutoFocusResetDelay" format="float"/>
```

Comment:
Can you use `integer|reference` here? I think that an int will be enough like we do for other attrs